test/index.test.js

```diff
@@ -279,6 +280,24 @@ describe('Segment.io', function() {
         analytics.assert(object.context);
         analytics.assert(!object.context.amp);
       });
+
+      it('should add a list of enabled integrations when `addEnabledMetadata` is set', function() {
```

Comment:
Please also include a negative test; i.e. `it should not add a list of enabled entegrations when addEnabledMetadata is unset`